tasks/generate-extension-imports.js

```diff
@@ -0,0 +1,44 @@
+var fs = require('fs');
+var path = require('path');
+var lstatSync = fs.lstatSync;
+
+var directories = fs.readdirSync(path.resolve(`${__dirname}/../scripts/extensions`)).filter(
+    (name) => lstatSync(path.resolve(`${__dirname}/../scripts/extensions/${name}`)).isDirectory()
+);
+
+const directoryNamingViolation = directories.find((name) => name.match(/^\w+$/g) == null);
+
+if (directoryNamingViolation != null) {
+    console.error(`"${directoryNamingViolation}" - extension directory names \
+must only contain alphanumerical characters and underscores`);
+    return;
+}
+
+let codeToImportExtensions =
+`
+/* tslint:disable */
+
+import {IExtensions} from 'superdesk-api';
+
+export const extensions: IExtensions = {};
+`;
+
+const importStatements = [];
+const insertIntoObjectStatements = [];
+
+directories.forEach((extensionName) => {
+    importStatements.push(`import * as ${extensionName} from '../extensions/${extensionName}/dist/extension';`);
```

Comment:
might be better to read `dist/extension` from package.json file rather than hardcoding it